ideascube/conf/base.py

```diff
@@ -105,27 +105,15 @@
 
 USE_TZ = True
 
-# Languages that will be available in all forms.
+# Languages that will be available as UI translations
 AVAILABLE_LANGUAGES = (
```

Comment:
Looking at it, moving `LANGUAGES` to `ideascube.conf.base` would mean moving also `EXTRA_LANG_INFO` to it.

That would remove from local configs the possibility to set there `EXTRA_LANG_INFO`. Do we want this?